lib/routes/account.js

```diff
@@ -531,6 +540,51 @@ module.exports = function (
             )
         }
 
+        function checkEmailAndPassword () {
+          // Session token verification is only enabled for certain users during phased rollout.
```

Comment:
Not my comment, it just looks like it since I had to re-organize the `readEmailRecord` function.